mindbender/maya/lib.py

```diff
@@ -166,6 +166,7 @@ def export_alembic(nodes,
                    file,
                    frame_range=None,
                    uv_write=True,
+                   write_Visibility=True,
```

Comment:
Use all lowercase please, so as to conform to the surrounding naming convention - a.k.a. snake_case.